Source/Resources/Content/AadIam/CompanyBrandings/Default.json

```diff
@@ -1,5 +1,6 @@
 {
   "bannerLogoUrl": "",
+  "description":"What is it: Defines the text and logos shown to users on Azure login screens. \n Simeon Notes: Provide your users with a personalized login screen for added security and familiarity. \n Learn More: https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/customize-branding",
```

Comment:
**What does this do?**
Defines the text and logos shown to users on Azure login screens. The baseline automatically populates the username watermark with 'user@companyname.org'. 

**Why should you use this?**
Provide your users with a personalized login screen for added security and familiarity.

**What is the end user impact?**
Users will see the watermark on Azure login screens.  

**Learn more**
[Customize your Azure AD sign-in page](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/customize-branding)

Reply:
@david-simeon  Can you see if we are actually changing anything with this file? Looking at it again, I don't know that we are

Reply:
@metamenos by default its not configured at all, so what we do is add a suggested username

Reply:
@david-simeon  ohh by water mark you mean, Username hint.

Reply:
@metamenos yeah, just added parentheses to make that more clear